pylot/simulation/perfect_tracker_operator.py

```diff
@@ -112,6 +112,11 @@ def on_notification(self, msg):
                                                       vehicle.id,
                                                       cur_vehicle_trajectory))
 
+        # Hack to also send the absolute position of vehicle
+        vehicle_trajectories.append(ObjTrajectory('vehicle',
+                                                  -1,
```

Comment:
Do you need to send output with vehicle id -1? Can't you send it with the real ego vehicle id? I've noticed that you have access to the ego vehicle id in the chauffeur_logger_operator.

Reply:
I could send with the real ego vehicle ID, but that ID is also used for the past poses. Using -1 makes it easier to determine which is the global transform vs. what is a past pose. 

Let me know if this is okay with you.